ocaml-lsp-server/src/compl.ml

```diff
@@ -208,15 +208,26 @@ module Complete_with_construct = struct
     | Error e -> Exn_with_backtrace.reraise e
     | Ok (loc, constructed_exprs) ->
       let range = Range.of_loc loc in
+      let deparen_constr_expr expr =
+        if
+          (not (String.equal expr "()"))
+          && Char.equal expr.[0] '('
+          && Char.equal expr.[String.length expr - 1] ')'
```

Comment:
How about:

```ocaml
if String.is_prefix expr ~prefix:"(" && String.is_suffix expr ~suffix:")" then
  String.sub expr ~pos:1 ~len:(String.length expr - 2)
else
  expr
```
This way you have 1 less conditional and you don't need to think about the empty string case.